TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -1227,7 +1227,7 @@ fileControlEntry:
 // p130: If the file connector referenced by file-name-1 is an external file connector, all file-control entries in the run unit that reference this file connector must have the same specification for the OPTIONAL phrase.
 
 selectClause:
-    SELECT OPTIONAL? fileNameDefinition;
+    SELECT OPTIONAL? fileNameReference;
```

Comment:
After second thought I think we can keep `fileNameDefinition` here.
Because, if later we want a SymbolTable for file declarations we will need a "SymbolDefinition".

Then for `fileDescriptionEntry` we could use a combination of reference and definition:
```suggestion
    (FD | SD) fileNameReferenceAndDataNameDefinition
```
Defined like that:
`fileNameReferenceAndDataNameDefinition: UserDefinedWord;`

Class `FileDescriptionEntry` still inherits from `DataDefinitionEntry` and has 2 properties:
- public SymbolReference FileName { get; set; }
- public SymbolDefinition DataName { get; set; };
What do you think ?

Reply:
I swapped between reference and definition because I think we should have a single instance of `SymbolDefinition` for the same thing. If we split into 2 definitions, one for the `FileName` and one for the `DataName` it is consistent. However, in order to compile, those two names must be iddentical right ? Where should this be checked ?

Reply:
Done